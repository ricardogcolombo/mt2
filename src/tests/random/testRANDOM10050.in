100 50
1 21 90 30 1
1 13 5 51 86
1 24 35 47 33
1 5 24 69 81
1 99 61 3 27
1 29 6 73 52
1 76 20 94 80
1 100 36 49 109
1 68 23 19 17
1 10 42 63 112
1 5 115 9 30
1 91 11 86 69
1 34 122 96 50
1 66 28 90 15
1 25 125 38 118
1 65 57 46 7
1 14 130 64 93
1 11 84 21 70
1 13 80 21 105
1 81 89 42 31
1 87 118 42 43
1 67 46 35 94
1 62 66 90 125
1 10 88 83 77
1 71 95 12 17
1 35 10 62 75
1 59 95 84 26
1 74 121 78 83
1 66 60 57 23
1 53 40 76 32
1 53 0 94 1
1 81 40 66 120
1 83 117 62 49
1 78 117 63 5
1 17 33 41 96
1 53 12 86 76
1 5 16 9 28
1 7 43 90 130
1 25 16 33 128
1 99 64 6 88
1 38 38 68 82
1 45 113 21 86
1 67 86 19 124
1 37 53 73 29
1 99 103 49 127
1 27 68 32 3
1 46 114 23 108
1 71 1 65 63
1 44 3 10 126
1 31 19 2 127
