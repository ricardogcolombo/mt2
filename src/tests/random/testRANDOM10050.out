0.500000
0.625000
0.310945
0.500000
0.409091
0.644279
0.372881
0.500000
0.454545
0.577512
0.662393
0.342105
0.269231
0.625000
0.500000
0.500000
0.375000
0.500000
0.478728
0.500000
0.487179
0.500000
0.342105
0.625000
0.489779
0.500000
0.625000
0.500000
0.333333
0.329060
0.375000
0.375000
0.663260
0.625000
0.406335
0.500000
0.666667
0.295856
0.500000
0.500000
0.625000
0.279238
0.500000
0.359171
0.662393
0.526316
0.375000
0.500000
0.708955
0.500000
0.589744
0.500000
0.432836
0.500000
0.500000
0.500000
0.438751
0.500000
0.625000
0.500000
0.500000
0.404075
0.532991
0.375000
0.763158
0.816253
0.221266
0.693646
0.636364
0.500000
0.526316
0.500000
0.500000
0.684817
0.500000
0.277612
0.500000
0.554452
0.500000
0.500000
0.523873
0.500000
0.495397
0.375000
0.500000
0.708955
0.593079
0.500000
0.500000
0.618642
0.402985
0.500000
0.500000
0.677612
0.500000
0.375000
0.500000
0.500000
0.432836
0.402985
